lib/SPIRV/OCLToSPIRV.cpp

```diff
@@ -742,56 +742,28 @@ void OCLToSPIRVBase::visitCallConvert(CallInst *CI, StringRef MangledName,
   if (auto *VecTy = dyn_cast<VectorType>(SrcTy))
     SrcTy = VecTy->getElementType();
   auto IsTargetInt = isa<IntegerType>(TargetTy);
-  auto TargetSigned = DemangledName[8] != 'u';
 
-  std::string TargetTyName(
+  std::string ConversionFunc(
       DemangledName.substr(strlen(kOCLBuiltinName::ConvertPrefix)));
-  auto FirstUnderscoreLoc = TargetTyName.find('_');
-  if (FirstUnderscoreLoc != std::string::npos)
-    TargetTyName = TargetTyName.substr(0, FirstUnderscoreLoc);
 
-  // Validate target type name
-  std::regex Expr("([a-z]+)([0-9]*)$");
+  // Validate target type name and vector size if present
```

Comment:
Maybe update this comment as follows to align with rename in code?
```suggestion
  // Validate conversion function name and vector size if present
```